lib/Bailador/Configuration.pm

```diff
@@ -134,8 +134,13 @@ class Bailador::Configuration {
                 self.load-from-hash(%config);
                 return;
             }
-            warn 'Error while loading the YAML config file.';
-            warn 'Bailador will use his default configuration.';
+            warn qq:to/WARNING/;
+            Error while loading the YAML config file.
+            Bailador will use his default configuration :
+                Mode : production
```

Comment:
shoudn't we fetch those values from the config object, if possible.
this is redundant information.

Reply:
yep, it was my initial thought, but I don't know if it's possible to do it easily. Or we can use a new variable....
```perl6
my $mode = 'production';
class Bailador::Configuration {
      has Str $.mode is rw where "production"|"development" = $mode;
}
warn $mode;
```